sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/jackson/SdaObjectMapperConfiguration.java

```diff
@@ -16,12 +16,19 @@
 import java.time.ZonedDateTime;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
 import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 
 @AutoConfiguration
+@Order(Ordered.HIGHEST_PRECEDENCE)
 public class SdaObjectMapperConfiguration {
-  @Bean
-  Jackson2ObjectMapperBuilder objectMapperBuilder() {
+  public static final String OBJECT_MAPPER_BUILDER_BEAN_NAME = "sdaObjectMapperBuilder";
+
+  @Bean(name = OBJECT_MAPPER_BUILDER_BEAN_NAME)
+  @Primary
+  Jackson2ObjectMapperBuilder sdaObjectMapperBuilder() {
```

Comment:
Should bean methods not be `public`?

Reply:
Changed. Not sure if needed (tests are passing), but feels better to make them public.